server/eclipse-project/src/main/resources/log4j.properties

```diff
@@ -1,15 +1,14 @@
 log4j.rootLogger=INFO, stdout
 
+#log4j.logger.us.freeandfair.corla.asm.AbstractStateMachine=DEBUG
 log4j.logger.us.freeandfair.corla.controller.ComparisonAuditController=DEBUG
 log4j.logger.us.freeandfair.corla.controller.BallotSelection=DEBUG
+#log4j.logger.us.freeandfair.corla.csv.DominionCVRExportParser=DEBUG
```

Comment:
Off by default, but now we know that we might be able to learn some fun things if we turn it on.